features/homepage/viewing_cart_items_count.feature

```diff
@@ -0,0 +1,18 @@
+@homepage
+Feature: Viewing a number of items in a cart
+    In order to track a number of items I have in my cart
+    As a Customer
+    I want to easily track a number of items in the cart
```

Comment:
In order to **track a number**... I want to **track a number**... 🧈 🧈 Also `As a Customer` is for the logged-in user, there should be `As a Visitor`

How about
```
In order to easily determine the number of products I'm about to buy
As a Visitor
I want to track the total quantity of the cart
```
?